e2e/blockstorage/resource_block_storage.go

```diff
@@ -155,7 +169,12 @@ func resourceUpdateBlockStorage(ctx context.Context, d *schema.ResourceData, m i
 	blockStorageID := d.Id()
 	project_id := d.Get("project_id").(int)
 	location := d.Get("location").(string)
+	status := d.Get("status").(string)
 
+	if status == "ERROR" {
+		rollbackChanges(d)
+		return diag.Errorf("Block Storage is in ERROR state. Cannot be updated. please destroy and recreate")
```

Comment:
please change the error message we can not ask the customer to delete and recreate the resource

Reply:
done